samples/dnn/tf_inception.cpp

```diff
@@ -78,12 +78,7 @@ int main(int argc, char **argv)
         exit(-1);
     }
 
-    cv::Size inputImgSize = cv::Size(224, 224);
-
-    if (inputImgSize != img.size())
-        resize(img, img, inputImgSize);       //Resize image to input size
-
-    Mat inputBlob = blobFromImage(img);   //Convert Mat to image batch
+    Mat inputBlob = blobFromImage(img, 1.0f, Size(224, 224));   //Convert Mat to batch of images
```

Comment:
`crop = false` so we have to add an empty `Scalar()` for means and `swapRB=true`